block-storage/src/main/scala/coop/rchain/blockstorage/ApprovedStore.scala

```diff
@@ -0,0 +1,20 @@
+package coop.rchain.blockstorage
+
+import cats.effect.Sync
+import cats.syntax.all._
+import coop.rchain.shared.syntax._
+import coop.rchain.store.{KeyValueStoreManager, KeyValueTypedStore}
+import scodec.bits.ByteVector
+import scodec.codecs._
+
+object approvedStore {
+  type ApprovedStore[F[_]] = KeyValueTypedStore[F, Byte, ByteVector]
```

Comment:
Type of value should not be raw ByteVector but ApprovedBlock so all encoding should be done in scodec definition. In the same way as BlockStore with BlockMessage.

Reply:
Probably [done](https://github.com/rchain/rchain/pull/3655/commits/099836b2cf7c151f85b598f7b004eed7b7c8b848).